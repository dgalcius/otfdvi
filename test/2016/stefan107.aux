\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\zref@newlabel[2]{}
\catcode `"\active 
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\CHKlabelout{chkflt.undef.0}{sc:scattering}
\refout{eq:time-dep}
\refout{eq:time-dep}
\refout{sc:scatt-equiv}
\CHKlabelout{chkflt.undef.0}{ss:scattering}
\CHKlabelout{chkflt.undef.0}{sc:scatter}
\refout{ss:limit}
\refout{eq:8.60}
\refout{eq:property-A}
\refout{eq:property-B}
\HyPL@Entry{0<</S/D /St 161>>}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_all}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_front}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_body}
\vktinfo{kt-ltx-chk}{cmn}{cmn_vln_all}
\vktinfo{kt-ltx-chk}{cmn}{cmn_vln_ext}
\vktinfo{kt-ltx-chk}{ltx}{ltx_dg_body}
\vktinfo{kt-ltx-chk}{xml}{xml_dg_all}
\vktinfo{kt-ltx-chk}{ltx}{ltx_dg_biblio}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_preamble}
\gdef\make@mdfive@files{stefan107.aux,stefan107.out,stefan107.idx,stefan107.pos}
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\@writefile{toc}{\contentsline {chapter}{\numberline {7}Scattering}{161}{stefan107.chapter.7:1}}
\@writefile{lof}{\addvspace {13\p@ }}
\@writefile{lot}{\addvspace {13\p@ }}
\newlabel{sc:scattering}{{7}{161}{Scattering}{stefan107.chapter.7:1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {7.1}Scattering operators}{161}{stefan107.section.1:2}}
\newlabel{ss:scattering}{{7.1}{161}{Scattering operators}{stefan107.section.1:2}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1.1}Physical meaning of $S$-operator}{161}{stefan107.subsection.1:3}}
\newlabel{sc:scatter}{{7.1.1}{161}{Physical meaning of $S$-operator}{stefan107.subsection.1:3}{}}
\refout{eq:8.63}
\refout{eq:8.63}
\refout{eq:8.64}
\refout{ss:adiaba}
\CHKlabelout{chkflt.undef.0}{lb:s-oper}
\CHKlabelout{chkflt.undef.1}{SC@1}
\CHKlabelout{chkflt.figure.1}{fig:6.1}
\refout{fig:6.1}
\newlabel{eq:8.63}{{7.1}{162}{Physical meaning of $S$-operator}{stefan107.equation.6}{}}
\newlabel{eq:8.64}{{7.2}{162}{Physical meaning of $S$-operator}{stefan107.equation.7}{}}
\newlabel{eq:8.65}{{7.3}{162}{Physical meaning of $S$-operator}{stefan107.equation.9}{}}
\@writefile{default}{\indexentry{S-operator|hyperpage}{162}}
\newlabel{lb:s-oper}{{7.1.1}{162}{Physical meaning of $S$-operator}{stefan107.equation.9}{}}
\newlabel{eq:8.73a}{{7.4}{162}{Physical meaning of $S$-operator}{stefan107.equation.10}{}}
\newlabel{eq:seta}{{7.5}{162}{Physical meaning of $S$-operator}{stefan107.equation.11}{}}
\citation{Goldberger,Taylor}
\citation{book,Kazes}
\newlabel{SC@1}{{\caption@xref {??}{ on input line 195}}{163}{Physical meaning of $S$-operator}{stefan107.figure.caption.12}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {Figure 7.1}{\ignorespaces Schematic representation of the scattering process.\relax }}{163}{stefan107.figure.caption.12}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig:6.1}{{7.1}{163}{Schematic representation of the scattering process.\relax }{stefan107.figure.caption.12}{}}
\newlabel{eq:8.62}{{7.6}{163}{Physical meaning of $S$-operator}{stefan107.equation.13}{}}
\@writefile{default}{\indexentry{S-matrix|hyperpage}{163}}
\@writefile{default}{\indexentry{scattering |hyperpage}{163}}
\refout{eq:8.65}
\CHKlabelout{chkflt.undef.1}{ss:perturbation}
\refout{eq:seta}
\refout{eq:8.67}
\refout{eq:8.67}
\refout{eq:8.66}
\newlabel{eq:S-rel-inv}{{7.7}{164}{Physical meaning of $S$-operator}{stefan107.equation.14}{}}
\newlabel{eq:8.65a}{{7.8}{164}{Physical meaning of $S$-operator}{stefan107.equation.15}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1.2}$S$-operator in perturbation theory}{164}{stefan107.subsection.2:16}}
\newlabel{ss:perturbation}{{7.1.2}{164}{$S$-operator in perturbation theory}{stefan107.subsection.2:16}{}}
\newlabel{eq:8.66}{{7.9}{164}{$S$-operator in perturbation theory}{stefan107.equation.17}{}}
\newlabel{eq:8.67}{{7.10}{164}{$S$-operator in perturbation theory}{stefan107.equation.19}{}}
\@writefile{default}{\indexentry{t-dependence|hyperpage}{164}}
\refout{eq:8.68}
\refout{eq:8.68}
\CHKlabelout{chkflt.undef.1}{ss:convenient}
\CHKlabelout{chkflt.undef.1}{lb:underline}
\CHKlabelout{chkflt.undef.1}{lb:underbrace}
\refout{eq:8.68}
\refout{eq:8.69}
\refout{eq:8.70}
\citation{book,Peskin}
\@writefile{default}{\indexentry{Dyson series|hyperpage}{165}}
\newlabel{eq:8.68}{{7.11}{165}{$S$-operator in perturbation theory}{stefan107.equation.20}{}}
\@writefile{default}{\indexentry{order of perturbation theory|hyperpage}{165}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1.3}Convenient notation for $t$-integrals}{165}{stefan107.subsection.3:21}}
\newlabel{ss:convenient}{{7.1.3}{165}{Convenient notation for $t$-integrals}{stefan107.subsection.3:21}{}}
\newlabel{lb:underline}{{7.1.3}{165}{Convenient notation for $t$-integrals}{stefan107.subsection.3:21}{}}
\newlabel{lb:underbrace}{{7.1.3}{165}{Convenient notation for $t$-integrals}{stefan107.subsection.3:21}{}}
\newlabel{eq:underline}{{7.12}{165}{Convenient notation for $t$-integrals}{stefan107.equation.22}{}}
\newlabel{eq:underbrace}{{7.13}{165}{Convenient notation for $t$-integrals}{stefan107.equation.23}{}}
\newlabel{eq:8.69}{{7.14}{165}{Convenient notation for $t$-integrals}{stefan107.equation.24}{}}
\newlabel{eq:8.70}{{7.15}{165}{Convenient notation for $t$-integrals}{stefan107.equation.25}{}}
\citation{Magnus,Magnus2,Blanes08}
\CHKlabelout{chkflt.undef.1}{lb:phi-oper}
\refout{eq:8.71}
\refout{eq:8.70}
\refout{eq:8.72}
\@writefile{default}{\indexentry{time-ordered series|hyperpage}{166}}
\newlabel{eq:F-D}{{7.17}{166}{Convenient notation for $t$-integrals}{stefan107.equation.28}{}}
\@writefile{default}{\indexentry{Magnus series|hyperpage}{166}}
\newlabel{eq:8.71}{{7.18}{166}{Convenient notation for $t$-integrals}{stefan107.equation.29}{}}
\newlabel{lb:phi-oper}{{7.1.3}{166}{Convenient notation for $t$-integrals}{stefan107.equation.29}{}}
\@writefile{default}{\indexentry{scattering phase operator|hyperpage}{166}}
\newlabel{eq:8.72}{{7.19}{166}{Convenient notation for $t$-integrals}{stefan107.equation.30}{}}
\@writefile{default}{\indexentry{time ordering|hyperpage}{166}}
\newlabel{eq:time-order}{{7.16}{166}{}{stefan107.equation.27}{}}
\@writefile{lpc}{\contentsline {lpcsec}{\numberline {}Original Text:\ - Inserted Text:}{166}{stefan107.Hfootnote.31}}
\CHKlabelout{chkflt.undef.1}{ss:adiaba}
\refout{eq:8.70}
\refout{eq:8.72}
\refout{sc:scatter}
\refout{ss:limit}
\newlabel{eq:7.63c}{{7.20}{167}{Convenient notation for $t$-integrals}{stefan107.equation.32}{}}
\newlabel{eq:7.63b}{{7.21}{167}{Convenient notation for $t$-integrals}{stefan107.equation.33}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1.4}Adiabatic switching of interaction}{167}{stefan107.subsection.4:34}}
\newlabel{ss:adiaba}{{7.1.4}{167}{Adiabatic switching of interaction}{stefan107.subsection.4:34}{}}
\newlabel{eq:basis1}{{7.22}{167}{Adiabatic switching of interaction}{stefan107.equation.35}{}}
\newlabel{eq:basis2}{{7.23}{167}{Adiabatic switching of interaction}{stefan107.equation.36}{}}
\newlabel{eq:9.54}{{7.24}{167}{Adiabatic switching of interaction}{stefan107.equation.37}{}}
\@writefile{default}{\indexentry{scattering states|hyperpage}{167}}
\refout{eq:9.54}
\citation{Goldberger}
\refout{eq:9.55}
\refout{eq:9.55}
\refout{eq:9.54}
\refout{eq:8.68}
\refout{eq:time-evoll}
\newlabel{eq:9.55}{{7.25}{168}{Adiabatic switching of interaction}{stefan107.equation.39}{}}
\@writefile{default}{\indexentry{adiabatic switching|hyperpage}{168}}
\newlabel{eq:adiaba}{{7.26}{168}{Adiabatic switching of interaction}{stefan107.equation.40}{}}
\newlabel{eq:9.56}{{7.27}{168}{Adiabatic switching of interaction}{stefan107.equation.42}{}}
\refout{eq:8.65a}
\refout{eq:8.68}
\CHKlabelout{chkflt.undef.1}{ss:t-matrix}
\refout{eq:8.68}
\refout{eq:basis1}
\refout{eq:basis2}
\refout{eq:9.56}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1.5}$T$-matrix}{169}{stefan107.subsection.5:44}}
\newlabel{ss:t-matrix}{{7.1.5}{169}{$T$-matrix}{stefan107.subsection.5:44}{}}
\citation{Rescigno,Brown-Jackson,Korchin,Dubovyk}
\CHKlabelout{chkflt.undef.1}{ss:s-matr-bound}
\refout{eq:SofE}
\citation{Goldberger,Taylor}
\newlabel{eq:Snm}{{7.28}{170}{$T$-matrix}{stefan107.equation.47}{}}
\@writefile{default}{\indexentry{T-matrix|hyperpage}{170}}
\newlabel{eq:SofE}{{7.29}{170}{$T$-matrix}{stefan107.equation.49}{}}
\refout{eq:SofE}
\refout{eq:SofE}
\refout{eq:6.71a}
\refout{eq:eine}
\CHKlabelout{chkflt.undef.1}{sc:scatt-equiv}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1.6}$S$-matrix and bound states}{171}{stefan107.subsection.6:50}}
\newlabel{ss:s-matr-bound}{{7.1.6}{171}{$S$-matrix and bound states}{stefan107.subsection.6:50}{}}
\newlabel{eq:eine}{{7.30}{171}{$S$-matrix and bound states}{stefan107.equation.51}{}}
\CHKlabelout{chkflt.undef.1}{ss:scatt-equiv}
\refout{eq:8.65a}
\refout{eq:8.69}
\refout{eq:F-D}
\refout{eq:8.71}
\@writefile{toc}{\contentsline {section}{\numberline {7.2}Scattering equivalence}{172}{stefan107.section.2:53}}
\newlabel{sc:scatt-equiv}{{7.2}{172}{Scattering equivalence}{stefan107.section.2:53}{}}
\@writefile{lpc}{\contentsline {lpcsec}{\numberline {}Original Text:\ - Inserted Text:}{172}{stefan107.section.2:53}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.2.1}Equivalent Hamiltonians}{172}{stefan107.subsection.1:54}}
\newlabel{ss:scatt-equiv}{{7.2.1}{172}{Equivalent Hamiltonians}{stefan107.subsection.1:54}{}}
\newlabel{eq:7.34a}{{7.31}{172}{Equivalent Hamiltonians}{stefan107.equation.55}{}}
\refout{eq:seta}
\citation{Ekstein}
\refout{eq:6.101b}
\refout{eq:8.75}
\refout{LemmaA.12}
\refout{eq:8.61a}
\CHKlabelout{chkflt.undef.1}{bakam-point}
\refout{ss:form-equiv-point}
\citation{Bakamjian}
\refout{ss:bakamjian}
\@writefile{default}{\indexentry{scattering equivalence|hyperpage}{173}}
\newlabel{eq:8.75}{{7.32}{173}{Equivalent Hamiltonians}{stefan107.equation.57}{}}
\newlabel{eq:6.101b}{{7.33}{173}{Equivalent Hamiltonians}{stefan107.equation.59}{}}
\newlabel{eq:6.101a}{{7.34}{173}{Equivalent Hamiltonians}{stefan107.equation.60}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.2.2}Bakamjian construction of point-form dynamics}{173}{stefan107.subsection.2:61}}
\newlabel{bakam-point}{{7.2.2}{173}{Bakamjian construction of point-form dynamics}{stefan107.subsection.2:61}{}}
\refout{eq:8.24}
\refout{eq:8.25}
\refout{ss:bakamjian}
\refout{eq:mass-mod}
\CHKlabelout{chkflt.undef.1}{ss:form-equiv-point2}
\refout{ss:scatt-equiv}
\refout{eq:7.34a}
\citation{Sokolov_Shatnii,Sokolov_Shatnii2}
\citation{Sokolov_Shatnii}
\newlabel{eq:mass-mod}{{7.35}{174}{Bakamjian construction of point-form dynamics}{stefan107.equation.62}{}}
\newlabel{eq:6.3.5}{{7.36}{174}{Bakamjian construction of point-form dynamics}{stefan107.equation.64}{}}
\@writefile{lpc}{\contentsline {lpcsec}{\numberline {}Original Text:\ - Inserted Text:}{174}{stefan107.equation.64}}
\refout{bakam-point}
\refout{eq:4.51a}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.2.3}Unitary link between point and instant forms of dynamics}{175}{stefan107.subsection.3:65}}
\newlabel{ss:form-equiv-point2}{{7.2.3}{175}{Unitary link between point and instant forms of dynamics}{stefan107.subsection.3:65}{}}
\CHKlabelout{chkflt.undef.1}{ss:form-equiv-point}
\refout{eq:8.73a}
\citation{Dollard}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.2.4}Scattering equivalence of forms of dynamics}{176}{stefan107.subsection.4:66}}
\newlabel{ss:form-equiv-point}{{7.2.4}{176}{Scattering equivalence of forms of dynamics}{stefan107.subsection.4:66}{}}
\@writefile{default}{\indexentry{M\"{o}ller wave operators|hyperpage}{176}}
\@writefile{default}{\indexentry{Birman--Kato invariance principle|hyperpage}{176}}
\refout{eq:S-rel-inv}
\citation{Sokolov_Shatnii,Sokolov_Shatnii2}
\newlabel{LastPage}{{}{177}{}{page.177}{}}
\xdef\lastpage@lastpage{177}
\xdef\lastpage@lastpageHy{177}
