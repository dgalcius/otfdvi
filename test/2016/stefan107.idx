\indexentry {S-operator|hyperpage}{162}
\indexentry {S-matrix|hyperpage}{163}
\indexentry {scattering |hyperpage}{163}
\indexentry {t-dependence|hyperpage}{164}
\indexentry {Dyson series|hyperpage}{165}
\indexentry {order of perturbation theory|hyperpage}{165}
\indexentry {time-ordered series|hyperpage}{166}
\indexentry {Magnus series|hyperpage}{166}
\indexentry {scattering phase operator|hyperpage}{166}
\indexentry {time ordering|hyperpage}{166}
\indexentry {scattering states|hyperpage}{167}
\indexentry {adiabatic switching|hyperpage}{168}
\indexentry {T-matrix|hyperpage}{170}
\indexentry {scattering equivalence|hyperpage}{173}
\indexentry {M\"{o}ller wave operators|hyperpage}{176}
\indexentry {Birman--Kato invariance principle|hyperpage}{176}
