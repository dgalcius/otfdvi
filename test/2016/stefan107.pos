\relax 
\vtexsavepos@pdf {figure1}{5594051sp}{11454148sp}{162}
