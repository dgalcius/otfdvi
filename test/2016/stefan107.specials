mt:proof
vtex:info.runtime.maker={pdfwf_luatex}
vtex:info.runtime.distribution={vtex}
vtex:info.runtime.format={2016}
vtex:info.runtime.cmdline={vtex 2016 pdfwf_luatex print stefan107.tex print -interaction=batchmode}
vtex:info.runtime.publisher={DGBOOK}
vtex:info.runtime.project={STEFAN1}
vtex:info.runtime.manuscript={STEFAN107}
vtex:info.runtime.historyone={2018.04.09 14:38}
vtex:info.runtime.docstage={PROOF}
vtex:info.runtime.pdftype={print}
vtex:info.runtime.outputdir={print}
