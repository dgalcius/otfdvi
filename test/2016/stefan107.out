\BOOKMARK [0][-]{stefan107.chapter.7:1}{\376\377\0007\000\040\000S\000c\000a\000t\000t\000e\000r\000i\000n\000g}{}% 1
\BOOKMARK [1][-]{stefan107.section.1:2}{\376\377\0007\000.\0001\000\040\000S\000c\000a\000t\000t\000e\000r\000i\000n\000g\000\040\000o\000p\000e\000r\000a\000t\000o\000r\000s}{stefan107.chapter.7:1}% 2
\BOOKMARK [2][-]{stefan107.subsection.1:3}{\376\377\0007\000.\0001\000.\0001\000\040\000P\000h\000y\000s\000i\000c\000a\000l\000\040\000m\000e\000a\000n\000i\000n\000g\000\040\000o\000f\000\040\000S\000-\000o\000p\000e\000r\000a\000t\000o\000r}{stefan107.section.1:2}% 3
\BOOKMARK [2][-]{stefan107.subsection.2:16}{\376\377\0007\000.\0001\000.\0002\000\040\000S\000-\000o\000p\000e\000r\000a\000t\000o\000r\000\040\000i\000n\000\040\000p\000e\000r\000t\000u\000r\000b\000a\000t\000i\000o\000n\000\040\000t\000h\000e\000o\000r\000y}{stefan107.section.1:2}% 4
\BOOKMARK [2][-]{stefan107.subsection.3:21}{\376\377\0007\000.\0001\000.\0003\000\040\000C\000o\000n\000v\000e\000n\000i\000e\000n\000t\000\040\000n\000o\000t\000a\000t\000i\000o\000n\000\040\000f\000o\000r\000\040\000t\000-\000i\000n\000t\000e\000g\000r\000a\000l\000s}{stefan107.section.1:2}% 5
\BOOKMARK [2][-]{stefan107.subsection.4:34}{\376\377\0007\000.\0001\000.\0004\000\040\000A\000d\000i\000a\000b\000a\000t\000i\000c\000\040\000s\000w\000i\000t\000c\000h\000i\000n\000g\000\040\000o\000f\000\040\000i\000n\000t\000e\000r\000a\000c\000t\000i\000o\000n}{stefan107.section.1:2}% 6
\BOOKMARK [2][-]{stefan107.subsection.5:44}{\376\377\0007\000.\0001\000.\0005\000\040\000T\000-\000m\000a\000t\000r\000i\000x}{stefan107.section.1:2}% 7
\BOOKMARK [2][-]{stefan107.subsection.6:50}{\376\377\0007\000.\0001\000.\0006\000\040\000S\000-\000m\000a\000t\000r\000i\000x\000\040\000a\000n\000d\000\040\000b\000o\000u\000n\000d\000\040\000s\000t\000a\000t\000e\000s}{stefan107.section.1:2}% 8
\BOOKMARK [1][-]{stefan107.section.2:53}{\376\377\0007\000.\0002\000\040\000S\000c\000a\000t\000t\000e\000r\000i\000n\000g\000\040\000e\000q\000u\000i\000v\000a\000l\000e\000n\000c\000e}{stefan107.chapter.7:1}% 9
\BOOKMARK [2][-]{stefan107.subsection.1:54}{\376\377\0007\000.\0002\000.\0001\000\040\000E\000q\000u\000i\000v\000a\000l\000e\000n\000t\000\040\000H\000a\000m\000i\000l\000t\000o\000n\000i\000a\000n\000s}{stefan107.section.2:53}% 10
\BOOKMARK [2][-]{stefan107.subsection.2:61}{\376\377\0007\000.\0002\000.\0002\000\040\000B\000a\000k\000a\000m\000j\000i\000a\000n\000\040\000c\000o\000n\000s\000t\000r\000u\000c\000t\000i\000o\000n\000\040\000o\000f\000\040\000p\000o\000i\000n\000t\000-\000f\000o\000r\000m\000\040\000d\000y\000n\000a\000m\000i\000c\000s}{stefan107.section.2:53}% 11
\BOOKMARK [2][-]{stefan107.subsection.3:65}{\376\377\0007\000.\0002\000.\0003\000\040\000U\000n\000i\000t\000a\000r\000y\000\040\000l\000i\000n\000k\000\040\000b\000e\000t\000w\000e\000e\000n\000\040\000p\000o\000i\000n\000t\000\040\000a\000n\000d\000\040\000i\000n\000s\000t\000a\000n\000t\000\040\000f\000o\000r\000m\000s\000\040\000o\000f\000\040\000d\000y\000n\000a\000m\000i\000c\000s}{stefan107.section.2:53}% 12
\BOOKMARK [2][-]{stefan107.subsection.4:66}{\376\377\0007\000.\0002\000.\0004\000\040\000S\000c\000a\000t\000t\000e\000r\000i\000n\000g\000\040\000e\000q\000u\000i\000v\000a\000l\000e\000n\000c\000e\000\040\000o\000f\000\040\000f\000o\000r\000m\000s\000\040\000o\000f\000\040\000d\000y\000n\000a\000m\000i\000c\000s}{stefan107.section.2:53}% 13
